api/rds.proto

```diff
@@ -586,6 +586,9 @@ message RedirectAction {
   // The HTTP status code to use in the redirect response. The default response
   // code is MOVED_PERMANENTLY (301).
   RedirectResponseCode response_code = 3 [(validate.rules).enum.defined_only = true];
+
+  // [#not-implemented-hide:] The scheme portion of the URL will be swapped with "https".
+  google.protobuf.BoolValue https_redirect = 4;
```

Comment:
Since this is default `false`, can you make the field type just a `bool`, thanks?